### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by potatoqualitee**
Woo! Just need to investigate adding built-in support for archiving

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,239 @@
...
+
  Helps Twitter communities find members on Mastodon. Searches lists, hashtags, account followers and more for Mastodon links and exports them to an importable CSV file.
+
+ This CSV file can be updated and published on a regular basis for other to import.
+
+> **Note**
+>
+> Because this is intended to help Twitter communities find their people on Mastodon, the default is to exclude any accounts set to private so the CSV files are built using purely public information.
+>
+> Examples below will outline how to include private accounts in the CSV output. This is useful when `exodus` to export Maston CSV files only for yourself.
```
Comment (thedavecarroll) [inline README.md:11]: Typo: Mastson should be Mastodon.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,239 @@
...
+        BLUEBIRDPS_API_SECRET: "${{ secrets.BLUEBIRDPS_API_SECRET }}"
+        BLUEBIRDPS_ACCESS_TOKEN: "${{ secrets.BLUEBIRDPS_ACCESS_TOKEN }}"
+        BLUEBIRDPS_ACCESS_TOKEN_SECRET: "${{ secrets.BLUEBIRDPS_ACCESS_TOKEN_SECRET }}"
+        BLUEBIRDPS_BEARER_TOKEN: "${{ secrets.BLUEBIRDPS_BEARER_TOKEN }}"
+```
+
+
+# Usage
+
+## Pre-requisites
```
Comment (thedavecarroll) [inline README.md:35]: Subjective typo. I think there is no dash (even though I have used a dash myself).

## Prerequisites

File: .github/workflows/seek.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        id: export
+        uses: potatoqualitee/exodus@initial
+        with:
+          hashtags: "powerbi,powerbicommunity,pbifamily"
+        env:
+          BLUEBIRDPS_API_KEY: "${{ secrets.BLUEBIRDPS_API_KEY }}"
+          BLUEBIRDPS_API_SECRET: "${{ secrets.BLUEBIRDPS_API_SECRET }}"
+          BLUEBIRDPS_ACCESS_TOKEN: "${{ secrets.BLUEBIRDPS_ACCESS_TOKEN }}"
+          BLUEBIRDPS_ACCESS_TOKEN_SECRET: "${{ secrets.BLUEBIRDPS_ACCESS_TOKEN_SECRET }}"
+          BluebirdPS_BEARER_TOKEN: "${{ secrets.BluebirdPS_BEARER_TOKEN }}"
```
Comment (justinjbird) [inline .github/workflows/seek.yml:18]: prob can remove this one

File: README.md [hunk 4]
```diff
@@ -1,2 +1,239 @@
...
+
+### Get some Twitter API keys
+
+This project uses [BluebirdPS](https://github.com/thedavecarroll/BluebirdPS) and Twitter API keys are required. If you don't have API access yet, Justin Bird wrote an [awesome tutorial](https://www.justinjbird.me/2022/how-to-set-up-twitter-developer-api-for-BluebirdPS/) to help you get going.
```
[the remark this answers is not in the document]
Comment (potatoqualitee) [reply README.md:38]: thank you! fixed

**Pull request merged**